src/main/java/StringCalculator.java

```diff
@@ -4,8 +4,11 @@
 
     public static final String EMPTY_STRING = "";
 
-    public int Add(String numbers) {
-        String[] arrayOfNumbers = numbers.split(",");
+    public int Add(String numbers) throws NumberFormatException {
+        if(numbers.chars().anyMatch(Character::isLetter)){
```

Comment:
we are filtering only letters, that mean input: "!@#$%^&*" will be not filtered.
The requirements was that we should accept only commas and numbers. allMatchMethod with another predicate will be suitable here. 
Of course testcase proposed above should be included in tests :)